packages/gcc_tools.rb

```diff
@@ -17,16 +17,19 @@ def self.build
 
 # define constants
 CREW_PREFIX="${CREW_PREFIX:-/usr/local}"
-ARCH="$(uname -m)"
+ARCH_ACTUAL="$(uname -m)"
+[[ "$ARCH_ACTUAL" == "armv8l" ]] && ARCH_ACTUAL="armv7l"
```

Comment:
Does this only happen with Docker images?  Most Chrome(books/boxes) (all?) return `aarch64` from the `arch/uname -m` command.

Reply:
armv8l is what shows up when you run ```linux32 uname -m``` on the raspberry pi. 

Otherwise you get aarch64.

The arm64 chromebooks give aarch64 from ```uname -m``` then?

And do they put libs in /usr/lib or /usr/lib64?

Reply:
The only architecture that has lib64 is x86_64.  All arm64/armv8/aarch64 architectures still use a 32-bit userland compatible with armv7l.

Reply:
Noted.

Reply:
took out this logic for the most part...